### ISSUE TITLE
Everything hashing at 0 during benchmark

### ISSUE THREAD

**Issue by roadh0use** [aaronsace/MultiPoolMiner#140]
I dont have a dump file unfortunately because I raged after spending 4 hours last night trying to get it to work and deleted everything. Now that I have gotten some sleep I'm a little calmer. 

Specs:
6 1060 3gb
64gb ssd
4gb ram
biostar t250-btc mobo
some intel celeron cpu

I have had claymore running for the past month on the rig with no issues dual mining eth and sc. Before that I was using ccminer to mine vertcoin. I know the system itself is functional.  Hell I tried it on my desktop computer as well and had the same results as follows:

Last night I ran the powershell file I typed in my username on miningpoolhub and hit enter as the instructions say. It starts, downloads some things, and starts benchmarking. The first one across all 6 cards says 0 sols. When i look at afterburners graph none of the gpus are under load at the time of benchmark. Then, ususally on the second algo's benchmark 9/10 the whole system freezes and I have to force restart. The 1/10 times it just does 0 hashs with no load on the gpu.

I have tried redownloading like 50 times both excavator and multipoolminer. I have installed the vcdistro related to the dll files you have listed. Always the same result. I was going to tear my hair out last night. So I just exited out after I saw 0 sols for the 100th time, opened back claymore and it's been hashing since 2am est without a hiccup.

If you need a log I will go out in the garage and rerun multipoolminer and upload the log file. Any help will be appreciated.

**Comment by aaronsace** [comment]
It's usually a sign that there is something wrong with your system. 6 cards is always pushing things to the limits. Common issues are with power and riser problems. 

I recommend narrowing down the miners and algorithms to ones that work best.

**Comment by roadh0use** [comment]
I'm going to go out there in a bit and reseat the risers and make sure everything is nice and tight. However, wouldn't the problem of 0 hashes persist on other standalone miners if it was a hardware issue. And also like I said even on my desktop inside, running multipoolminer on it yeilds the same results and that is just a single gpu with no overclock on cpu or gpu.

**Comment by roadh0use** [comment]
Alright I have reseated the risers and still the same.

HOWEVER

I tried something different. I downloaded the master and added claymore in there because it was lacking it. Running the updated master has made the 0 hash problem disappear. It seems to be working now. The only issue is there is a lot of red text but it is actually benchmarking so I'll take it. I guess this problem can be marked as solved.
